Packages/CommonUI/Sources/CommonUI/View/TextField/PasscodeTextField.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by đông on 17/03/2022.
+//
+
+import SwiftUI
+
+private enum Constants {
+	static let textFieldHeight = 52.0
+	static let paddingVertical = 14.0
+	static let paddingHorizontal = 24.0
+	static let spacing = 14.0
+	static let widthPasscode = 56.0
+}
+
+public struct PasscodeTextField: View {
+	// MARK: - Variables
+	@Binding var text: String
+	@Binding var inputStyle: TextInputStyle
+	private let keyboardType: UIKeyboardType
+
+	// MARK: - Init
+	public init(text: Binding<String>,
+				inputStyle: Binding<TextInputStyle>,
+				keyboardType: UIKeyboardType = .numberPad) {
+		self._text = text
+		self._inputStyle = inputStyle
+		self.keyboardType = keyboardType
+	}
+    var body: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/90062f405b1acb14bb3deb33ce97c24ecf5b8748/Packages/CommonUI/Sources/CommonUI/View/TextField/PasscodeTextField.swift#L32"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>PasscodeTextField.swift:32</code></td>
    </tr>
  </tbody>
</table>